x-pack/plugins/cloud_security_posture/server/routes/benchmark_rules/find/v2.ts

```diff
@@ -11,9 +11,9 @@ import { CSP_BENCHMARK_RULE_SAVED_OBJECT_TYPE } from '../../../../common/constan
 
 import type {
   CspBenchmarkRule,
-  FindCspBenchmarkRuleRequest,
   FindCspBenchmarkRuleResponse,
 } from '../../../../common/types/latest';
+import type { FindCspBenchmarkRuleRequest } from '../../../../common/types/rules/v4';
```

Comment:
` CspBenchmarkRule` and `FindCspBenchmarkRuleResponse` also should be imported from v4.

Reply:
CspBenchmarkRule are not being used here tho